### DISCUSSION TITLE
Crashing on PC

### DISCUSSION THREAD

**Discussion by Somnorite** [thatskymod/Sky-CotL-Scripts#145] [Q&A] [answered]
I think the game autoupdated and its crashing everytime i load TSM, if anyone drops an update of it pls lmk

**Comment by thatskymod** [comment, marked as answer]
Hello,

I apologize for the outage. That said, any time the base game is updated, mods also need to be aligned with the new logic. This may take some time.

Consider trying the newly released ThatSkyMod [v3.7.0](https://github.com/XeTrinityz/ThatSkyMod/releases/tag/3.7.0). This version should resolve any outstanding issues found in its predecessors.
